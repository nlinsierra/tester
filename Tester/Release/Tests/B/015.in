9H TD 6D 4S 2H 7S 6H QS QD 3H KC 4C AS 4D 8H JS 5C 3S 3D QC 2C KS 3C 5H 8D JC JH 7C AH 6S 4H 2S AC 9D 2D QH TH 8S
