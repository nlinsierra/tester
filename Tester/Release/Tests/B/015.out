14
6C 8C 9C TC 5D 7D JD KD AD 7H KH 5S 9S TS
